_pages/schedule.md

```diff
@@ -22,3 +22,17 @@ The conference will take place from 9AM to 6PM.
 
 <a class="button" href="{{ site.schedule_link }}" target="_blank">View schedule!</a>
 
+## Sprints
+
+So you have listened to all the exciting topics on Django for three days and now feel motivated do something about it. What could be a better motivation than contributing to the very open source software you just listened to?
+
+On Thursday and Friday, you have the opportunity to do exactly that. This is your opportunity to contribute to Open Source software. Even if you haven't contributed before, this is also an option to take your first steps. In the sprint, you can contribute to Django as you see fit, and if you need help, there will be a pool of experienced contributors to guide you. You can also reach out through slack in the sprints channel.
```

Comment:
```suggestion
On Thursday and Friday, you have the opportunity to do exactly that. This is your opportunity to contribute to Open Source software. Even if you haven't contributed before, this is also an option to take your first steps. In the sprint, you can contribute to Django projects as you see fit, and if you need help, there will be a pool of experienced contributors to guide you. You can also reach out through Discord in the sprints channel.
```
I don't know for sure that there will be people sprinting on Django itself

Reply:
What will they be doing? I've never been to one

Reply:
@rwillmer Sorry, I missed this question. Sprints are sessions of people working together on open source projects (or docs, in some cases).  The actual projects will be proposed at the conference, usually by people who are the main maintainer(s). I think often Django core developers will sprint on Django itself at a Django Con, but I don't want to say that on the website without knowing it for sure.

There's an intro talk I've seen Daniele do - I can't find the video right now, but this is a good description, which we should probably steal and put in the pretalx schedule:
https://pretalx.com/pyconuk-2019/talk/K973NJ/